README.md

```diff
@@ -120,10 +120,13 @@ ashDumpPersistenceSchemaOutputDirectoryPath := "~" // Changes directory
 
 ### 3. Serializer
 
-[Circe-based](https://circe.github.io/circe/) Akka serializer. It uses Circe codecs, derived using Magnolia, that are
```

Comment:
We don't even have Magnolia on the classpath AFAICS 🤔

Reply:
Circe comes from, ekhm, different side of the _Great Divide_ in the Scala community, they're unlikely to have picked Magnolia as their default choice

Reply:
(although there's some half-maintained https://github.com/vpavkin/circe-magnolia)

Reply:
Nice catch, did not notice it.